### ISSUE TITLE
Error when attempting get request with Node 18.7

### ISSUE THREAD

**Issue by dhatcher** [m0rtadelo/ntlm-client#5]
I'm attempting to test the following call in my environment (updating the necessary fields):

```
const response = await client.request('http://ntlm.protected.data/items?id=26',
  'user', 'pass', 'workstation', 'domain'
);
```
When I make the call I get the following error:

**Uncaught Error: error:0308010C:digital envelope routines::unsupported**

Stepping through the stack trace the it appears the issue happens when a hash is being created for the password.

I've looked online and it seems like something changed with Node in version 17 that causes this problem.

**Comment by m0rtadelo** [comment]
i cannot reproduce the error. I use the next code:

```
const NtlmClient = require('node-client-ntlm').NtlmClient;
const client = new NtlmClient();
test();

async function test() {
  const response = await client.request('http://ntlm.protected.data/items?id=26',
    'user', 'pass', 'workstation', 'domain',
  );
}

```
result:
```
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: getaddrinfo ENOTFOUND ntlm.protected.data
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26) {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'ntlm.protected.data'
}

Node.js v18.13.0
```

**Comment by m0rtadelo** [comment]
Fixed in v0.0.6
